autograder/submitty_autograding_shipper.py

```diff
@@ -853,16 +854,20 @@ def checkout_vcs_repo(config, my_file):
 
         # is vcs_subdirectory standalone or should it be combined with base_url?
         elif vcs_subdirectory[0] == '/' or '://' in vcs_subdirectory:
-            vcs_path = vcs_subdirectory
+            if '.git' in vcs_base_url:
```

Comment:
Would it be better to add a 'subfolder grading' variable to the gradeable config instead of checking if it is an external Git repository ? If so, could someone advise me on how to do that ? I am not very versed in the config sides of Submitty